index.html

```diff
@@ -174,12 +175,32 @@ <h2>About</h2>
             <li>GitLab</li>
             <li>Terminal</li>
           </ul>
-          <img src="images/ic_skills.svg" class="svg-image3 position" alt="svg image">
+          <img src="images/ic_skills.svg" class="about-images" alt="svg image">
         </div>
       </div>
     </div>
   </section>
 
+  <form action="https://formspree.io/f/xwkaywbg" class="contact-form d-flex flex-cols" method="POST">
+    <h2 class="typography primary-color">Get Started</h2>
+    <div class="cta-summary poppins"><p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+    </p>
+  </div>
+    <span class="d-flex flex-cols input-wrapper margin-tp">
+      <label for="name" class="poppins">Your name</label>
+      <input class="primary-color no-outline" type="text" name="name" id="name" placeholder="" title="Your name" maxlength="30" required>
+    </span>
+    <span class="d-flex flex-cols input-wrapper">
+      <label for="email" class="poppins">Your email</label>
+      <input class="primary-color no-outline" type="text" name="email" id="email" placeholder="" title="Your email" required>
+    </span>
```

Comment:
- [ ] It will be good if the input tag for the email has a type of email not text, so there won't be invalid submissions of email.

Reply:
- [ ] The placeholder attribute for both input tags are not being used, I suggest that it should be removed.